CONTRIBUTING.adoc

```diff
@@ -20,7 +53,7 @@ Enable the `Eclipse code formatter` by clicking `Use the Eclipse code formatter`
 * IDEA's "Optimize imports on the fly" option turned on interferes with the Eclipse code formatter import optimization.
 Consider disabling the option if optimization does not yield the expected results.
 
-= Contributing a patch
+== Contributing a patch
 
 1. Submit an issue describing your proposed change
```

Comment:
2. After working out any ambiguities, create a pull request with the proposed code change.